### PR TITLE
PTNFLY-526 toast notification

### PR DESCRIPTION
**Pull request by andresgalante**
@rhamilto can you review it?
This adds the toast notification.
Please take a close look at how I've generated the arrow on the drop menu, and the names of the classes for max width and positioning.
Thanks!

### CODE REVIEW

File: tests-src/toast.html [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+---
+      <div class="toast-pf alert alert-success">
+        <div class="pull-right toast-pf-action">
+          <a href="#">Start Server</a>
+        </div>
+        <span class="pficon pficon-ok"></span>
+        <strong>server</strong> has been added to <a href="#" class="alert-link">main server</a>.
+      </div>
+      <br>
+      <div class="toast-pf alert alert-warning">
```
Comment (rhamilto) [inline tests-src/toast.html:15]: Replace this with {% include toast.html %} so this toast notification isn't duplicated?

**Comment by dtaylor113** [comment]
LGTM

File: less/toast.less [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+      font-size: (@font-size-base + 4);
+      line-height: 1;
+      padding: 13px 5px 13px 5px;
+      &:active,
+      &:focus,
+      &:hover {
+        color: @link-color;
+      }
+    }
+    .dropdown-menu{
```
Comment (rhamilto) [inline less/toast.less]: Could use a space between "u" and "{'.

File: less/toast.less [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+      right: -15px;
+      &:before{
```
Comment (rhamilto) [inline less/toast.less]: Could use a space between "e" and "{".

File: less/toast.less [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+  > .pficon {
+    background-color: @gray-pf;
+    bottom:  -1px;
+    box-shadow: 2px 0 5px -2px rgba(0,0,0,.2);
+    left: -1px;
+    padding-top: 10px;
+    text-align: center;  
+    top: -1px;
+    width: 53px;
+    &:before{ color: hsla(0, 0%, 100%, .74); }
```
Comment (rhamilto) [inline less/toast.less]: Could use a space between "e" and "{".

File: less/toast.less [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+  }
+  .toast-pf-action{
```
Comment (rhamilto) [inline less/toast.less]: Could use a space between "n" and "{".

File: less/toast.less [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+    margin-left: 15px;
+  }
+  /* Medium devices (desktops, 992px and up) */
+  @media (min-width: @screen-md-min) {
+    display: inline-block;
+  }
+}
+// To set max width on toast notifications
+.toast-pf-max-width{
```
Comment (rhamilto) [inline less/toast.less]: Could use a space between "h" and "{".

File: less/toast.less [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+ @media (min-width: @screen-md-min) {
+    max-width: 31.1% ; // sets the size to 4 cols;
+  }
+}
+// To position the notification on the top right corner, used in context on cards layout
+.toast-pf-top-right{
```
Comment (rhamilto) [inline less/toast.less]: Could use a space between "t" and "{".

File: dist/css/patternfly-additions.css [hunk 8]
```diff
@@ -4220,6 +4220,103 @@ table.datatable th:active {
...
+  .toast-pf {
+    display: inline-block;
+  }
+}
+.toast-pf-top-right {
+  left: 20px;
+  position: absolute;
+  right: 20px;
+  top: 12px;
+  z-index: 1070;
```
Comment (rhamilto) [inline dist/css/patternfly-additions.css]: Use a var from https://github.com/twbs/bootstrap/blob/master/less/variables.less#L265?  Should toasts overlay modals?  If not, this value should be less than https://github.com/twbs/bootstrap/blob/master/less/variables.less#L277

File: less/toast.less (hunk 4 above)
Comment (rhamilto) [inline less/toast.less]: Bump ^

**Comment by rhamilto** [comment]
A couple other things:

Please delete the .DS_Store you've added.  We should add that to .gitignore.

Can you also squash your commits?

**Comment by andresgalante** [comment]
@rhamilto I'll add  .DS_Store to .gitignore and try to squash your commits :)

**Pull request merged**